rollup.config.js

```diff
@@ -36,6 +36,7 @@ const config = {
     nodeResolve({mainField: ['jsnext:main']}),
   ].concat(compress ? terser() : []),
   output,
+  inlineDynamicImports: true,
```

Comment:
I haven't looked into why this is now required to build. My best guess is the optional React import in utils.